go/libkb/pseudonym.go

```diff
@@ -260,3 +258,12 @@ func checkTlfPseudonymFromServer(ctx context.Context, g *GlobalContext, req TlfP
 
 	return nil
 }
+
+func RandomHmacKey() [32]byte {
+	key := [32]byte{}
+	n, err := cryptorand.Read(key[:])
```

Comment:
you can call `RandBytes`.  If you want, you can add a check in there too to make sure we're getting the right number of bytes back out.